### DISCUSSION TITLE
Keyball39 / 一番下の段が反応しない..

### DISCUSSION THREAD

**Discussion by daseu** [Yowkees/keyball#637] [Q&A]
大変お世話になります。

再度の投稿、失礼いたします..。
先般ご指摘いただいた問題を修正したところ、
おかげさまで大部分が動作するようになりました。本当にありがとうございました。

組み立てRemapにてキー割当てをしテストをしていたのですが、
一番下の段がすべて反応しないようです..。（Remapにてキー変更）
どのようなことが原因として考えられますでしょうか..?

お知恵を拝借できますと幸いです。
何卒、よろしくお願いいたします。

![100のコピー](https://github.com/user-attachments/assets/08645bb0-28bd-47aa-bcae-b63decd25467)
![110](https://github.com/user-attachments/assets/867c523a-22e5-4810-bdc4-c607c59b899b)
![115](https://github.com/user-attachments/assets/ffba2c10-3812-46fe-b467-88a903deae3a)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

左右共最下段が不良というのは、偶然だと思いますので、ひとまず片手ずつ半田付けの見直しをしていただければと思います。
ProMicro裏の”ROW3”と刻印がある端子が隣とショートしていないかご確認ください。
また、下画像緑丸の部分も、ショートしていると最下段のキーに影響しますので確認いただければ幸いです。

![short](https://github.com/user-attachments/assets/04838723-bae5-4beb-8acb-ae421c49afcd)

以上、お手数ですがご確認お願いいたします。

**Reply by daseu** [reply]
大変お世話になっております。

お返事が遅くなり申し訳ございません。

ご丁寧に画像マーキングでのご指摘をいただきありがとうございます。
取り急ぎのお礼でございます。
